pkg/tanka/load_test.go

```diff
@@ -107,6 +107,36 @@ func TestLoad(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:    "inline-filename",
+			baseDir: "./testdata/cases/withenv/main.jsonnet",
+			expected: manifest.List{{
+				"apiVersion": "v1",
+				"kind":       "ConfigMap",
+				"metadata": map[string]interface{}{
+					"name":      "config",
+					"namespace": "withenv",
+				},
+			}},
+			env: &v1alpha1.Environment{
+				APIVersion: v1alpha1.New().APIVersion,
+				Kind:       v1alpha1.New().Kind,
+				Metadata: v1alpha1.Metadata{
+					Name:      "withenv",
+					Namespace: "cases/withenv/main.jsonnet",
```

Comment:
This creates inconsistency, where I could receive a different namespace depending on whether I invoke my environment as `./cases/withenv` or `./cases/withenv/main.jsonnet`. This would trip up `prune` once `pruneMark` is merged.

To be consistent, we should probably always include the filename

Reply:
> To be consistent, we should probably always include the filename
Shall I change that here?

Reply:
see a990c8d